spec/api/controllers/players/show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+RSpec.describe Api::Controllers::Players::Show, type: :action do
+  let(:repository) { PlayerRepository.new }
+  let(:interactor) do
+    Interactors::Players::FetchPlayer.new(repository: repository)
+  end
+  let(:action) { described_class.new(interactor: interactor) }
+
+  context "the player doesn't exist" do
+    before { repository.clear }
+
+    it 'is unsuccessful' do
+      response = action.call(id: 1)
+      expect(response[0]).to eq(400)
+    end
+  end
+
+  context 'the player exists' do
+    before do
+      repository.clear
+      repository.create(id: '1')
+      repository.create(id: '2')
+    end
+
+    it 'is successful' do
+      response = action.call(id: 1)
+
+      expect(response[0]).to eq(200)
```

Comment:
What is `response[0]`? Hard to understand what we're testing here.

Reply:
@jgayfer fixed.